### PR TITLE
Update releasing runbook

### PR DESCRIPTION
**Pull request by sdb9696**

### CODE REVIEW

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]: I'd shorten this to three simple bullet points (given this is just for maintainers anyway):
```
## Required packages
* [github client](url)
* [gitchub_changelog_generator](url)
* [github access token](url)
```

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]:
```suggestion
```

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]:
```suggestion
export NEW_RELEASE=0.
```

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]:
```suggestion
```

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]: Perhaps we should do this always as part of the release process? This would make sure that the newest packages are then tested to work with the release build.

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]:
```suggestion
## Create release summary (skip for dev releases)

```

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]: Is this necessary?

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md]:
```suggestion
```
This is so obvious that we don't need to state it for ourselves :-)

**Review by rytilahti — CHANGES_REQUESTED** [review]
Looking great! Some comments, mainly about avoiding to add unnecessary information and keeping it short as this is just for us maintainers :-)

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: It feels a bit late in the day, i.e. we've been testing a bunch of code and then just before we release we update all the dependencies.  I think that should probably be a an earlier explicit PR we do in the lead up to a release.

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: I spent a little bit of time looking into the warnings.  This was to avoid anyone else having to do the same.

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: I don't know how cross-platform the regex is so it thought it prudent to stress to check.  Maybe I'll just add it as a comment to the `echo` bash line.

File: RELEASING.md
Comment (rytilahti) [reply RELEASING.md]: Ok, fair enough :+1:

File: RELEASING.md
Comment (rytilahti) [reply RELEASING.md]: I mean, it's a _prepare_ release PR after all and it just updates our own pins, so it should be fine.

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Maybe
```bash
export NEW_RELEASE=x.x.x.devx
```

File: RELEASING.md
Comment (rytilahti) [reply RELEASING.md]: Yeah, maybe it's a good idea to print out the variables prior doing steps that change something?

File: RELEASING.md
Comment (rytilahti) [reply RELEASING.md]: Also fine, even when it's quite obvious that this is something one should fill.

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Done

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md]: Removed and comment added to the `echo` bash line

File: RELEASING.md
Comment (rytilahti) [inline RELEASING.md:83]:
```suggestion
export EXCLUDE_TAGS=${NEW_RELEASE%.dev*}; export EXCLUDE_TAGS=${EXCLUDE_TAGS//"."/"\."}; export EXCLUDE_TAGS="^((?!"$EXCLUDE_TAGS")(.*dev\d))+"
```
These need to be exported.

**Review by rytilahti — APPROVED** [review]
LGTM after exporting the variables.

File: RELEASING.md
Comment (sdb9696) [reply RELEASING.md:83]: I don't think they do as they're not used by any sub processes

Comment (rytilahti) [reply RELEASING.md:83]: Ohhh, hmm, didn't think about it that way. So it's working as expected?

Comment (sdb9696) [reply RELEASING.md:83]: Yes, it works

**Pull request merged**
